client/editor/tool_tile.h

```diff
@@ -3,6 +3,9 @@
 
 #pragma once
 
+// generated
+#include <packets_gen.h>
```

Comment:
I think this should be `packets.h`...
```suggestion
#include "packets_gen.h"
```